server/schema/user/user.datasources.js

```diff
@@ -152,6 +152,9 @@ const create = async (uid, fullName, email, interestedTopics, session, authToken
 };
 
 // TODO: Update all redundancies
+
+const addStorePath = (uid, storePath) => admin.auth().updateUser(uid, { picture: { photoURL: storePath } });
```

Comment:
Rename this to `updateProfilePicture` and modify the function to support changing the full picture object.